cumulus/pallets/xcmp-queue/src/lib.rs

```diff
@@ -477,31 +377,40 @@ impl OutboundChannelDetails {
 
 #[derive(Copy, Clone, Eq, PartialEq, Encode, Decode, RuntimeDebug, TypeInfo)]
 pub struct QueueConfigData {
-	/// The number of pages of messages which must be in the queue for the other side to be told to
-	/// suspend their sending.
+	/// The number of pages which must be in the queue for the other side to be told to suspend
+	/// their sending.
 	suspend_threshold: u32,
-	/// The number of pages of messages which must be in the queue after which we drop any further
-	/// messages from the channel.
+	/// The number of pages which must be in the queue after which we drop any further messages
+	/// from the channel. This should normally not happen since the `suspend_threshold` can be used
+	/// to suspend the channel.
 	drop_threshold: u32,
-	/// The number of pages of messages which the queue must be reduced to before it signals that
+	/// The number of pages which the queue must be reduced to before it signals that
 	/// message sending may recommence after it has been suspended.
 	resume_threshold: u32,
-	/// The amount of remaining weight under which we stop processing messages.
+	/// UNUSED - The amount of remaining weight under which we stop processing messages.
+	#[deprecated(note = "Will be removed")]
```

Comment:
seems like we need a follow-up issue for little things like migrating this, removing the stale `MigrationState` etc.

Reply:
Yea one thing is done here https://github.com/paritytech/polkadot-sdk/pull/1836 but there are a few other follow ups.